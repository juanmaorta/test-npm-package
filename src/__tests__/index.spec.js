/* global expect, describe, it */
import React from 'react'
import { shallow } from 'enzyme'

import MyComponent from '../index'

describe('MyComponent snapshots', () => {
  it('should match for MyComponent component', () => {
    const wrapper = shallow(
      <MyComponent />
    )
    expect(wrapper).toMatchSnapshot()
  })
})
