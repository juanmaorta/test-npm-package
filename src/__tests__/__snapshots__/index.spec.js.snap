// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent snapshots should match for MyComponent component 1`] = `ShallowWrapper {}`;
